app/src/main/java/org/sopt/dosopttemplate/MainActivity.kt

```diff
@@ -2,10 +2,22 @@ package org.sopt.dosopttemplate
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-
+import org.sopt.dosopttemplate.databinding.MainPageBinding
 class MainActivity : AppCompatActivity() {
+    lateinit var binding: MainPageBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(R.layout.main_page)
+        binding = MainPageBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val id = intent.getStringExtra("id")
+        val mbti = intent.getStringExtra("mbti")
+        val nickname = intent.getStringExtra("nickname")
+
+        binding.yourId.text = "$id"
```

Comment:
```suggestion
        binding.yourId.text = id
```
이미 string 타입이니, 이렇게 작성해도 동일합니다!